### PR TITLE
chore: speed up N* build

### PR DESCRIPTION
**Pull request by layershifter**
## Problem 💣

On any change that has `@fluentui/docs` in a scope we are calling `build:component-info` **three times** (for a task that runs **2 minutes**) 🥵

```
2021-05-20T09:30:34.9876815Z verb @fluentui/react-northstar build:info |  Finished 'build:component-info' after 2.43 min
2021-05-20T09:30:35.0960481Z info @fluentui/react-northstar build:info ✔️ done  - 2m 34.82s

//  👆 is called for `@fluentui/react-northstar build` ✔

// ----

2021-05-20T09:33:37.7397940Z verb @fluentui/react-northstar build |  Finished 'build:component-info' after 1.87 min
2021-05-20T09:33:37.8619721Z info @fluentui/react-northstar build ✔️ done  - 2m 53.84s

//  👆 is called for `@fluentui/docs build` because of a config of Lage 🟥

// ----

2021-05-20T09:37:44.9203314Z verb @fluentui/docs build |  [09:35:48] Starting 'build:component-info'...
2021-05-20T09:37:44.9204870Z verb @fluentui/docs build |  [09:37:44] Finished 'build:component-info' after 1.93 min

// 👆 is called for `@fluentui/docs build` in Gulp scripts
//    - fine for local builds ✔
//    - bummer for CI 🟥
```

In fact, we should run it once.

### Solution

- remove an entry for Lage config as it's outdated
- remove `component-info` from `bundle` task (we will build info once in packages)

---

## Problem (smaller) 💣

We run `@fluentui/docs build` during "BuildTestLint" which has no sense 🙄
(that also runs `component-info` three times)

### Solution

Rename script in `@fluentui/docs` from `build` to `bundle` to not build it during "BuildTestLint".

## Problem (smallest) 💣

We call `build:info` during `build` this slow downs Size Auditor job & "BuildTestLint".

### Solution

Move "build:info" calls to "bundle" instead of "build" as we need them only for "Deploy".\

### Result

image
🔽
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
   "scripts": {
     "a11ytest": "cd apps && cd a11y-tests && yarn test",
     "build": "lage build --verbose",
-    "build:fluentui:docs": "gulp build:docs",
```
Comment (layershifter) [inline package.json:13]: We should use `yarn workspace` for this.

File: gulpfile.ts [hunk 2]
```diff
@@ -29,6 +24,3 @@ require('./scripts/gulp/tasks/perf');
 require('./scripts/gulp/tasks/test-e2e');
 require('./scripts/gulp/tasks/test-circulars');
 require('./scripts/gulp/tasks/test-dependencies');
-
-// global tasks
-task('build', parallel('build:docs'));
```
Comment (layershifter) [inline gulpfile.ts:34]: We don't use this anywhere.

File: gulpfile.ts [hunk 3]
```diff
@@ -1,8 +1,3 @@
-// *** UNDER HEAVY CONSTRUCTION ***
-// add / modify gulp tasks as your own peril - we're migrating to just for our build tasks!
-// expect gulp tasks to be converted to just within the next 2-3 weeks!
```
Comment (layershifter) [inline gulpfile.ts:3]: We don't plan to do this.

Comment (ecraig12345) [reply gulpfile.ts:3]: Heh...that comment survived for way too long!! 🤣

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**